package.json

```diff
@@ -47,7 +47,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "echo '🕺  Running eslint on staged files' && lint-staged --allow-empty && echo '🏃‍♂️  Running pretty-quick on staged files' && pretty-quick --staged"
+      "pre-commit": "echo '🕺 Running eslint on staged files' && lint-staged --allow-empty && echo '🏃‍♂️ Running pretty-quick on staged files' && pretty-quick --staged"
```

Comment:
unrelated change. spacing in the logs was a bit broken. previous:
![image](https://user-images.githubusercontent.com/12886084/90604145-b00ef580-e1fc-11ea-96d2-c417e211a9ce.png)

after change:
![image](https://user-images.githubusercontent.com/12886084/90604158-b604d680-e1fc-11ea-8ac5-6d7fc456c5b6.png)